packages/astro/src/vite-plugin-build-css/index.ts

```diff
@@ -0,0 +1,151 @@
+
+import type { ResolveIdHook, LoadHook, RenderedChunk } from 'rollup';
+import type { Plugin as VitePlugin } from 'vite';
+
+import { STYLE_EXTENSIONS } from '../core/ssr/css.js';
+import { getViteResolve, getViteLoad } from './resolve.js';
+import { getViteTransform, TransformHook } from '../vite-plugin-astro/styles.js';
+import * as path from 'path';
+
+
+const PLUGIN_NAME = '@astro/rollup-plugin-build-css';
```

Comment:
```suggestion
const PLUGIN_NAME = '@astrojs/rollup-plugin-build-css';
```

Reply:
Actually this should probably be `vite-plugin-build-css`, right? I think we’re using some Vite APIs in here.